hathor/p2p/node_sync.py

```diff
@@ -492,6 +495,10 @@ def next_step(self) -> Generator[Deferred, Any, None]:
             self.log.debug('already running')
             return
 
+        if not self.is_enabled:
```

Comment:
Just to fully understand this change, with this `return` this node (node A) won't be trying to find the synced timestamp with the other node (node B) but if B is still in the old version, it will continue working as it works today, right? Finding the synced timestamp and we will reply the messages even with `sync_enabled = False`. The high CPU usage happens in this method (next_step), so the new nodes will be fine with CPU use, is that correct?

Reply:
Your understand of the impact of this line is correct. But you're right that the other node might run the sync algorithm and it might have a high impact on the CPU. I'll fix it in a separate commit.